app/javascript/Search/__tests__/SearchForm.test.jsx

```diff
@@ -1,20 +0,0 @@
-import { h } from 'preact';
```

Comment:
This test was completely removed because `<SearchForm />` gets tested in the `<Search />` test.